src/NuGet.Core/NuGet.ProjectModel/Utf8JsonStreamReader.cs

```diff
@@ -220,6 +222,25 @@ internal bool ReadNextTokenAsBoolOrFalse()
             return false;
         }
 
+        internal bool ReadNextTokenAsBoolOrTrue(byte[] propertyName)
```

Comment:
I think your latest changes removed all usage of this method, in which case I think you should delete this new method.

Reply:
We still use it in [JsonPackageSpecReader.Utf8JsonStreamReader.cs](https://github.com/NuGet/NuGet.Client/pull/5833/files#diff-5e8ef6d8ff5b223efadc3ad1d0162c8e43a663827bf73f73af89e03ffab9d2a1). I think it should be renamed to a more appropriate name: `ReadNextTokenAsBoolOrThrowAnException`